vignettes/explanation-project-structure.Rmd

```diff
@@ -63,18 +63,29 @@ server <- function(id) {
 }
 ```
 
-## Minimal `app.R`
-Shiny sources `app.R` the traditional way, so `library()` and `source()` could be used.
-Box modules can only load other box modules,
-so by keeping the `app.R` minimal and loading the whole app with box
-we can be confident that it is properly used throughout the app.
-
-Additionally, `rhino:app()` gives us full control over app startup. It does:
-1. Purge box cache (reload app without restarting R)
-2. Configure logger (level, file)
-3. Configure static files
-4. Load main module / legacy entrypoint
-5. Add head tags (favicon, CSS & JS)
+# Minimal `app.R`
```

Comment:
I suggest a title like "Explanation: application structure in Rhino"

Reply:
Renamed to "Explanation: Application structure". I decided to leave "Rhino" out - I think its obvious.